.gitignore

```diff
@@ -0,0 +1,34 @@
+*/.gitignore
```

Comment:
same comment here to add some spacing between the sections per this example: https://github.com/android/compose-samples/find/main